packages/@okta/vuepress-site/docs/reference/api/idps/index.md

```diff
@@ -4420,10 +4422,9 @@ Content-Type: application/json
 
 ### Link a User to a social provider without a Transaction
 
-
 <ApiOperation method="POST" url="/api/v1/idps/${idpId}/users/${userId}" />
 
-Links an Okta User to an existing [social provider](#identity-provider-object). This endpoint doesn't support the SAML2 [Identity Provider type](#identity-provider-type).
+Links an Okta User to an existing [social provider](#identity-provider-object). To enable persistent account linking with a SAML Identity Provider, ensure that `honorPersistentNameId` is set to `true` and the [Name Identifier Format](/docs/reference/api/idps/#saml-2-0-settings-object) of the incoming assertion is `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent`.
```

Comment:
Links an Okta User to an existing SAML or social provider. The SAML Identity Provider must have `honorPersistentNameId` is set to `true` to be able to use this API. The [Name Identifier Format](/docs/reference/api/idps/#saml-2-0-settings-object) of the incoming assertion must be `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent` to honor this link.